public/index.php

```diff
@@ -1,14 +1,37 @@
 <?php
 
-declare(strict_types=1);
-
 use Tempest\Discovery\DiscoveryLocation;
 use Tempest\Router\HttpApplication;
+use Tempest\Router\WorkerApplication;
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
-HttpApplication::boot(__DIR__ . '/../', discoveryLocations: [
+$discoveryLocations = [
     new DiscoveryLocation('Tests\\Tempest\\Fixtures\\', __DIR__ . '/../tests/Fixtures/'),
-])->run();
+];
+
+if (function_exists('frankenphp_handle_request')) {
```

Comment:
This won't be enough, as this method exists even when FrankenPHP is running in non-worker mode, and the call below to this method will throw an exception.

This would be a proper check:

```suggestion
$isWorkerMode = $_SERVER['FRANKENPHP_WORKER'] ?? false;

if ($isWorkerMode && function_exists('frankenphp_handle_request')) {
```